plugins/inventory/foreman.py

```diff
@@ -169,10 +169,7 @@
   - name.split('.')[0]
 '''
 import json
-try:
-    from ansible.module_utils.compat.version import LooseVersion
-except ImportError:
-    from distutils.version import LooseVersion
```

Comment:
Is the vendored `LooseVersion` buggy? Given the careful fallback import i would not expect the new version to be less compatible than the old one.

Reply:
no, but ansible < 2.9.20 and < 2.10.8 is buggy when it comes to imports with try/except: https://github.com/theforeman/foreman-ansible-modules/issues/1352

Reply:
Oh wow. Are there 2.9.21 and 2.10.9 that fix it? I'd say supporting the latest z in every y would be acceptable.

Reply:
2.9.20 and 2.10.8 are the fixing versions, yes. versions prior to that flat out go belly up.

I am torn, as not every supported deployment platform of ours has these versions easily available :(

Reply:
Giving a sad ACK here.